format-as-currency.js

```diff
@@ -65,7 +65,9 @@ angular
       })
 
       ngModel.$parsers.push(function (value) {
-
+        // ignore non-numeric characters
+        value = value.replace(/[a-zA-Z!\?>:;\|<@#%\^&\*\)\(\+\/\\={}\[\]_]/g, '')
```

Comment:
btw the build fails because of coveralls, you're probably aware but figured I'd point it out.  All the tests pass but coveralls fails for some reason.